deepdiff/diff.py

```diff
@@ -94,6 +94,7 @@ def _report_progress(_stats, progress_logger, duration):
     'ignore_type_subclasses',
     'ignore_string_case',
     'exclude_obj_callback',
+    'exclude_obj_callback_strict',
```

Comment:
@mskhviyu 
I wonder if this should be passed to DeepHash or not. When doing deephash calculations, we don't have both of the objects. We have only one of them.

Here in your unit test, if we set the `exclude_obj_callback_strict` and also `ignore_order=True`, it should throw and exception since DeepHash does not accept that parameter.

Reply:
my mistake, I didn't want to add to deephash. Removed from DEEPHASH_PARAM_KEYS, now my unit test passes successfully with the ignore_order=True parameter